app/assets/javascripts/utils.js

```diff
@@ -31,6 +31,25 @@ function brightness(rgb) {
     return rgb.r * 0.299 + rgb.g * 0.587 + rgb.b * 0.114;
 }
 
+/**
```

Comment:
Include link to https://caniuse.com/#feat=download in comments. Basically IE11 only

_[Original comment](https://github.ugent.be/unipept/unipept/pull/625#discussion_r13725) by @bmesuere on Mon Feb 19 2018 at 14:29._